Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM node:16.3-slim
-RUN apt-get update && apt-get install git g++ make python tree -y
+FROM node:16.18.1-slim
```

Comment:
Do we need it to be pinned to a specific patch version or can we keep it pinned to a minor version like it was before?

Reply:
No, we don't have to. Changed to `16.18`.